modulefiles/hera.intel/fv3

```diff
@@ -58,7 +59,7 @@ module load hdf5_parallel/1.10.6
 ##
 ## load cmake
 ##
-module use /contrib/modulefiles
+module use /contrib/cmake/modulefiles
```

Comment:
Why not remove the `module use` line and just do
```
module load cmake/3.16.1
```
(available as part of the standard modules)?

Reply:
@climbfuji - I wasn't aware that cmake/3.16.1 had been installed on Hera. Assuming that it still works with the newer version, I agree it would make more sense to use it instead.

Reply:
> @climbfuji - I wasn't aware that cmake/3.16.1 had been installed on Hera. Assuming that it still works with the newer version, I agree it would make more sense to use it instead.

Not sure what the testing procedures are for the production branch, but I am sure it will work. We use it with `develop`.

Reply:
Ok. I pushed it.